aggie-coding-club/Rev-Registration
# Instruction type tooltips

## Description

Adding a tooltip to the meeting type under "section select".

## How to test

Hover over a meeting type and make sure that the abbreviation matches the tooltip

## Screenshots
### Before
<img width="328" alt="Screen Shot 2020-10-03 at 10 57 33 PM" src="https://user-images.githubusercontent.com/26681493/95006595-e8f80380-05cb-11eb-8a81-a602444dd392.png">

### After
<img width="331" alt="Screen Shot 2020-10-06 at 8 52 51 PM" src="https://user-images.githubusercontent.com/26681493/95278506-f3a5d900-0815-11eb-8eb8-77b83c48b8c8.png">

Closes #298
https://app.zenhub.com/workspaces/rr-5f735a6a098ca70016574cf1/issues/aggie-coding-club/rev-registration/298
